src/styles/block/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+
+  &__img {
+    margin-left: 3px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 36px;
+    margin-bottom: 4px;
+    font-weight: $font-weight-medium;
+    font-size: $font-size-normal;
+    line-height: 18px;
+    color: $color-dark;
+  }
+  
+  &__code {
+    font-size: $font-size-small;
+    margin: 0;
```

Comment:
If you want to reset the margins, you should do that in reset.scss. Same for all classes
```suggestion
```